autogluon/task/tabular_prediction/predictor.py

```diff
@@ -501,23 +501,32 @@ def feature_importance(self, model=None, dataset=None, features=None, raw=True,
 
         Parameters
         ----------
-        model : str, default = None
-            Model to get feature importances for, if None the best model is chosen.
-            Valid models are listed in this `predictor` by calling `predictor.model_names`
         dataset : str or :class:`TabularDataset` or `pandas.DataFrame` (optional)
             This dataset must also contain the label-column with the same column-name as specified during fit().
             If specified, then the dataset is used to calculate the feature importance scores.
             If str is passed, `dataset` will be loaded using the str value as the file path.
             If not specified, the original dataset used during fit() will be used if `cache_data=True`. Otherwise, an exception will be raised.
             Do not pass the training data through this argument, as the feature importance scores calculated will be inaccurate.
+        model : str, default = None
+            Model to get feature importances for, if None the best model is chosen.
+            Valid models are listed in this `predictor` by calling `predictor.model_names`
         features : list, default = None
             List of str feature names that feature importances are calculated for and returned, specify None to get all feature importances.
             If you only want to compute feature importances for some of the features, you can pass their names in as a list of str.
-        raw : bool, default = True
-            Whether to compute feature importance on raw features in the original data (after automated feature engineering) or on the features used by the particular model.
-            For example, a stacker model uses both the original features and the predictions of the lower-level models.
-            Note that for bagged models, feature importance calculation is not yet supported when both `raw=True` and `dataset=None`. Doing so will raise an exception.
-        subsample_size : int, default = 10000
+        feature_stage : str, default = 'original'
+            The stage of features to compute importances on.
```

Comment:
Can we clarify: "The stage of features to compute importances on"
 -> 
"What stage of feature-processing should importances be computed for."

Reply:
Added